aws/table_aws_codestar_notification_rule.go

```diff
@@ -0,0 +1,188 @@
+			{
+				Name:        "name",
+				Description: "The name of the notification rule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "resource",
+				Description: "The Amazon Resource Name (ARN) of the resource associated with the notification rule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Resource"),
+			},
+			{
+				Name:        "detail_type",
+				Description: "The level of detail included in the notifications for this resource. BASIC will include only the contents of the event as it would appear in Amazon CloudWatch. FULL will include any supplemental information provided by AWS CodeStar Notifications and/or the service for the resource for which the notification is created.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("DetailType"),
+			},
+			{
+				Name:        "status",
+				Description: "The status of the notification rule. Valid statuses are on (sending notifications) or off (not sending notifications).",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Status"),
+			},
+			{
+				Name:        "event_types",
+				Description: "A list of the event types associated with the notification rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("EventTypes"),
+			},
+			{
+				Name:        "targets",
+				Description: "A list of targets associated with the notification rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Targets"),
+			},
+			{
+				Name:        "created_by",
+				Description: "The name or email alias of the person who created the notification rule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("CreatedBy"),
+			},
+			{
+				Name:        "created_timestamp",
+				Description: "The date and time the notification rule was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("CreatedTimestamp"),
+			},
+			{
+				Name:        "last_modified_timestamp",
+				Description: "The date and time the notification rule was most recently updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("LastModifiedTimestamp"),
+			},
+			{
+				Name:        "tags_src",
+				Description: "The tags associated with the notification rule.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     describeNotificationRule,
+				Transform:   transform.FromField("Tags"),
+			},
+		}),
+	}
+}
```

Comment:
@gertvv I think the column definition is missing the standard columns like `akas`, `title` and `tags` columns.

Please refer https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_ec2_instance.go#L488-L506 for additional information.